blinkit-app/src/homePage/atoms/insertList/InsertList.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+
+class InsertList extends Component {
+  
+  render() {
+    const list = this.props.list;
+    return (
+      <>
+        {list.map((item, index) => {
```

Comment:
Do not use inline functions, it decreases the readability of a component, create a separate helper, which can also be reused if needed elsewhere

Reply:
Also atoms dont consist of helpers, if you need a helper for your component please create this as a molecult